weaviate/templates/weaviateStatefulset.yaml

```diff
@@ -269,8 +269,16 @@ spec:
               {{- end }}
             {{- end }}
           {{- end  }}
-          {{- if or (index .Values "backups" "s3" "enabled") (index .Values "modules" "text2vec-aws" "enabled") (index .Values "modules" "generative-aws" "enabled") }}
-            {{- if index .Values "backups" "s3" "envSecrets" }}
+          {{- if or (index .Values "offload" "s3" "enabled") (index .Values "backups" "s3" "enabled") (index .Values "modules" "text2vec-aws" "enabled") (index .Values "modules" "generative-aws" "enabled") }}
```

Comment:
I'm missing some code like this:
```
          {{- if index .Values "backups" "s3" "enabled" }}
             {{- if index .Values "backups" "s3" "envconfig" }}
               {{- range $key, $value := index .Values "backups" "s3" "envconfig" }}
```
So that the envconfig section from offload.s3 is passed to the environment variables.

Reply:
you set those `env` values like `AWS_SECRET_KEY` in the `env:` section bc it applies to all `AWS` modules